# H3-MONGODB
Je n'arrive pas à lancer le projet Python, j'ai un problème de connexion à la base de données Redis que j'ai lancé avec Docker.